.github/workflows/publish.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - name: set up JDK 11
         uses: actions/setup-java@v3
         with:
-          java-version: '11'
+          java-version: '17'
```

Comment:
1. Praise (❤️): Thanks for updating Java to 17! 🥇 
2. Minor (🔍): Consider changing the name to `set up JDK 17` as well.

PS: Same applies to the [readme.yml](https://github.com/Parsely/parsely-android/pull/77/files#diff-b6c03303a900e88f776648edf9f69e0e7370e7e6ac08b364e6dacc4bc1a5c3e7R16) such change.